package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "perspective-node",
```

Comment:
@iislucas I guess I'm not sure what you mean by "supporting other js clients." Do you mean other Perspective clients with differing functionality? Do you mean non-Node js, as in a browser environment?